bench/qemu-test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# Runs specified SSHFILE on QEMU guest and then kills buildroot
+
+ROOT=buildroot
+TERMINAL="gnome-terminal"
+
+# command line arguments to script with defaults
+HOST_SH="${1:-wiredtiger-copy.sh}" # file to run on host
+GUEST_SH="${2:-wiredtiger-run-ycsb-c5.sh}" # file to run on guest
+QEMU_MEM="${3:-8192}" # memory in MB of QEMU guest
+CORES="${4:-4}" # number of cores to emulate
+EXTRA_DRIVE="${5:-/media/jackson/WDC_HDD/disk_imgs/disk_120G.raw}" # other hard drive to add. TODO: once dev is complete, setup default so that it's not necessary
+TIMEOUT="${6:-0}" # in seconds
+
+TEST_PROGRAM=(${GUEST_SH//_/ })
+LOG_PREFIX="${TEST_PROGRAM[0]}_${QEMU_MEM}MB_"
+LOG_DIR="logs"
+
+STDOUT_LOG_SUFFIX=".log"
+STDOUT_LOG_NAME="$LOG_DIR/$LOG_PREFIX$(date '+%Y-%m-%d--%H-%M-%S')$STDOUT_LOG_SUFFIX"
+STDERR_LOG_SUFFIX=".err"
+STDERR_LOG_NAME="$LOG_DIR/$LOG_PREFIX$(date '+%Y-%m-%d--%H-%M-%S')$STDERR_LOG_SUFFIX"
+
+mkdir -p $LOG_DIR
+
+echo "QEMU guest memory (mb): $QEMU_MEM" >> $STDOUT_LOG_NAME
+echo "Test execution file: $GUEST_SH" >> $STDOUT_LOG_NAME
+
+#$TERMINAL --title="QEMU monitor" -- 
+timeout $TIMEOUT qemu-system-x86_64 -enable-kvm \
+ -kernel ../output/images/bzImage \
+ -initrd ../output/images/rootfs.cpio.gz \
+ -append "console=ttyS0" \
+ -cpu host \
+ -smp $CORES \
+ -m $QEMU_MEM \
+ -usb \
+ -chardev pty,id=ser0 \
+ -serial chardev:ser0 \
+ -net user,hostfwd=tcp::10022-:22 -net nic \
+ -drive file=../swap-10g.raw,format=raw,if=ide \
+ -drive file=$EXTRA_DRIVE,format=raw,if=ide & # TODO: allow for no extra drive
+
+# wait for QEMU to start, run script on host, ssh and run guest script.
+# TODO, wait on QEMU monitor (if possible) instead of sleeping
+sleep 10 && \
+bash $HOST_SH && \
+(cat $GUEST_SH | sshpass -p "root" ssh root@localhost \
+  -p 10022 \
+  -o "UserKnownHostsFile /dev/null" \
+  -o StrictHostKeyChecking=no) \
+>> $STDOUT_LOG_NAME 2> $STDERR_LOG_NAME &
+
+wait
+
+# there's a typo in wt monitor files
```

Comment:
Can you give me more details about this typo in wt? Did you look at submitting a patch upstream to fix it?

Reply:
The typo is a missing comma in the csv header which this adds in. I'll take a look at the WT source code and hopefully fix it there

Reply:
not 100% necessary, but if it's approximately the same effort for you to fix the WT bug, it will (a) help a lot more people and (b) make your code cleaner - no workaround.

Reply:
Might as well take a look at it. Would be cool to make a (single comma) contribution to WT

Reply:
will leave this in for now and take a look at WT